packages/visualizations-react/stories/Map/SvgChoropleth.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { SvgChoropleth } from '../../src';
+import { shapes } from './shapes';
+
+const scales = {
+    none: undefined,
+    blue: {
+        type: 'gradient',
+        colors: {
+            start: '#bcf5f9',
+            end: '#0229bf',
+        },
+    },
+};
+
+// I know data is plural already!
+const dataSets = {
+    1: {
+        value: [
+            { x: 'France', y: 60 },
+            { x: 'Île de France', y: 35 },
+            { x: 'Corsica', y: 95 },
+        ],
+    },
+    2: {
+        value: [
+            { x: 'France', y: 20 },
+            { x: 'Île de France', y: 75 },
+            { x: 'Corsica', y: 30 },
+        ],
+    },
+};
+
+const meta: Meta = {
+    title: 'Map/SvgChoropleth',
+    component: SvgChoropleth,
+};
+
+export default meta;
+const Template = ({ height, width, colorsScale, dataSet, options }) => {
+    const optionsWithScale = { colorsScale: scales[colorsScale], ...options };
+    console.log(dataSet, dataSets[dataSet]);
+    return (
+        <div
+            style={{
+                margin: 'auto',
+                border: '1px solid black',
+                padding: '13px',
+                display: 'inline-block',
+                height,
+                width,
+            }}
+        >
+            <SvgChoropleth
+                options={optionsWithScale}
+                data={dataSets[dataSet]}
+                style={{ height: '100%', width: '100%' }} // necessary to remove the wrapper div
```

Comment:
It's hard to understand why you need to pass this here, is it always needed ? I think it's a bit tricky to know that you need to add this here, can't it be a mandatory option of SvgChoropleth ?

Reply:
You mean the `style` part? It's because story book adds an empty wrapper div for some reason if there is no style. But it shouldn't be needed in any real code.

Reply:
Ah ok, wasn't aware of that. I've just tested removing it and the result was disturbing 😃 Maybe you can just add to the comment that this is a storybook specificity to be more explicit ?